Libplanet/Net/InvalidStateTargetException.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Runtime.Serialization;
+
+namespace Libplanet.Net
+{
+    [Serializable]
+    internal class InvalidStateTargetException : SwarmException
```

Comment:
As what “state target” means here is not obvious (out of context), how about making this an inner class of `GetRecentStates` and renaming it `GetRecentStates.InvalidTargetException`? This would be easier to get it because the term “invalid target” is common between <code>GetRecentStates.<em>InvalidTarget</em>Exception</code> class and <code>GetRecentStates.<em>InvalidTarget</em></code> property.

Reply:
Also it would be better if this class has a property named `InvalidTarget` (or more simply `Target`) property.